### DISCUSSION TITLE
UnrealBuildTool: ERROR in Windows 10

### DISCUSSION THREAD

**Discussion by Wasinee63070700312** [carla-simulator/carla#4165] [Q&A] [answered]
Good morning everyone

I have installed Carla in window10 but when I use command make launch it got the error as 

"UnrealBuildTool: ERROR: Unable to instantiate module 'Carla': System.IO.DirectoryNotFoundException: Could not find a part of the path 'C:\Users\koowa\Desktop\carla\Unreal\CarlaUE4\Plugins\Carla\CarlaDependencies\lib'."

What should I do? 
PS. I already downloads Asset and zin file already, I am using VS2017.
![image](https://user-images.githubusercontent.com/71056491/116792422-b32be680-aaea-11eb-935e-ccc9b1ed3f07.png)

**Comment by corkyw10** [comment, marked as answer]
Hi @Wasinee63070700312, what version of CARLA are you installing? Can you provide the documentation that you are following? Do you have 7zip installed? What version of Make do you have installed?

**Reply by Wasinee63070700312** [reply]
Hello @corkyw10  I can fix my problem. It because of when I downloads carla file form github, it has some file missing and i did not notice. So, I redownload and it can use.

Thank you for you kind :)

**Reply by snknitheesh** [reply]
can u please elaborate on this ? Which file from github ?

**Reply by Franncys** [reply]
I also had this issue, I went to the Carla dependencies folder and I saw that "lib" was a file instead of a folder. You can open it with 7zip and extract it into a new folder called lib. After that it worked.

![image](https://github.com/carla-simulator/carla/assets/16153497/3a7af812-5bc3-42d6-8c06-3d9c5ae4d1d0)

**Reply by Runningchauncey** [reply]
Just want to be more specific on this problem cuz it took me a while to find this.
The dependencies folder is at "carla\Unreal\CarlaUE4\Plugins\Carla\CarlaDependencies"

**Reply by alisaleem97** [reply]
> I also had this issue, I went to the Carla dependencies folder and I saw that "lib" was a file instead of a folder. You can open it with 7zip and extract it into a new folder called lib. After that it worked.
> 
> Please, Where I can find ( __lib, _lib ) to download, I need only theses files.

**Reply by Franncys** [reply]
sorry for the confusion but the __lib and _lib files were simply backups I did of the lib file. You should just extract the lib file.
